ci/prep.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -ex
+
+rel_file="/etc/os-release"
+
+common_pkgs=(
+  cmake
+  createrepo_c
+  e2fsprogs
+  expat-devel
+  findutils
+  gpgme-devel
+  libsolv-devel
+  openssl-devel
+  popt-devel
+  python3-devel
+  python3-pip
+  python3-pyOpenSSL
+  python3-requests
+  python3-setuptools
+  python3-urllib3
+  rpm-build
+  sed
+  sqlite-devel
+  sudo
+  util-linux
+  valgrind
+  which
+)
+
+if grep -qw "Fedora" ${rel_file}; then
+  fedora_packages=(
+    ${common_pkgs[@]}
+    gcc
+    glib2-devel
+    libcurl-devel
+    make
+    python3-pytest
+    python3-flake8
+    rpm-devel
+    rpm-sign
+    shadow-utils
+  )
+  dnf -y upgrade --refresh
+  dnf -y install ${fedora_packages[@]}
+elif grep -qw "Photon" ${rel_file}; then
+  photon_packages=(
+    ${common_pkgs[@]}
+    build-essential
+    curl-devel
+    glib
+    glibc-debuginfo
+    shadow
+    zlib-devel
+  )
+  tdnf -y upgrade --refresh
+  tdnf remove -y toybox
+  tdnf -y install --enablerepo=photon-debuginfo ${photon_packages[@]}
+  pip3 install pytest flake8
```

Comment:
We have flake in Photon:
```
okurth@photon-dev [ ~/poi-harness ]$  (main)$ tdnf search flake
python3-pyflakes : A simple program which checks Python source files for errors
okurth@photon-dev [ ~/poi-harness ]$  (main)$ tdnf repoquery --list python3-pyflakes
/usr/bin/pyflakes
```
Can we use that?

Reply:
They are not the same. From the pyflakes project page: https://github.com/PyCQA/pyflakes
```
If you require more options and more flexibility, you could give a look to [Flake8](https://pypi.org/project/flake8/) too.
```